Sources/StreamChat/APIClient/Endpoints/Payloads/ChannelListPayload.swift

```diff
@@ -94,6 +94,10 @@ struct ChannelDetailPayload<ExtraData: ExtraDataTypes>: Decodable {
     /// Refer to [docs](https://getstream.io/chat/docs/multi_tenant_chat/?language=swift) for more info.
     public let team: String
     
+    /// Cooldown duration for the channel, if it's in slow mode.
+    /// This value will be 0 if the channel is not in slow mode.
+    let cooldownDuration: Int
+    
```

Comment:
nit: I get this value is in whole seconds but still - maybe it's better to use `TimeInterval` for this?

Reply:
backend only accepts seconds, if we use `TimeInterval` users who skip documentation will assume it's ms...